app/models/eligibilities/v3/exhibit.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Eligibilities
+  module V3
+    # Exhibit is a source of information that acts as a proof to support an evidence (or a raised Data Matching Inconsistency).
+    # Exhibit can be:
+    #   - a document uploaded by the user that is verified by the admin
+    #   - a call can be made to an external (DMV, IRS, SSA, etc) service to verify the evidence
+    #   - admin can manually verify the evidence
+    #   - self-attested by the user in some cases (for example, American Indian Status)
+    class Exhibit
+      include Mongoid::Document
+      include Mongoid::Timestamps
+      include ::HasDocument
```

Comment:
I don't think top level exhibit needs this has_document as the only type that involves documents is the upload_exhibit